source/sass/blocks/form-group.scss

```
@@ -59,18 +82,43 @@
 
 .form-group__input {
   flex-grow: 1;
-  padding: 10px;
+  padding: 14px 12px;
   font-family: $added-regular-font;
   font-size: 16px;
   line-height: 18px;
   font-weight: 400;
   color: $color-grey-700;
   border: 1px solid $color-grey-500;
   text-transform: uppercase;
+
+  &[type="email"] {
+    background-image: url("../img/icons/icon_email.svg");
+    background-repeat: no-repeat;
+    background-position: 92% 50%;
```

Лучше сделай позиционирование по горизонтали в пикселях, а то чем шире, тем больше отступ от правого края (проценты же).
Если что, для фона можно прописывать расстояния от конкретных сторон:
```suggestion
    background-position: right 20px top 50%;
```